### ISSUE TITLE
ERROR: While executing gem ... (errno: : ENOENT) 

### ISSUE THREAD

**Issue by Kasperait** [rubygems/rubygems#2421]
I'm having a problem or would like to suggest a feature.

My current problem is When I try to set ruby's settings.rubidium. Writes me an error no such file or directory @ dir_chdir - bundler/exe

This issue is related to:

  - [ ] Network problems
  - [x] Installing a library
  - [ ] Publishing a library
  - [ ] The command line gem
  - [ ] Other

Here are my current environment details: `ruby setup.rb`

```
$ gem env version
PASTE HERE
```

I will abide by the [code of conduct](https://github.com/rubygems/rubygems/blob/master/CODE_OF_CONDUCT.md).

**Comment by bronzdoc** [comment]
I don't understand how this is a Rubygems issue? please provide more details of the error to be able to help you, i'm going to close this for now.

**Comment by Kasperait** [comment]
Okay I'll try to explain in more detail, plus I'll attach a photo. When installing rubygems I originally wrote (lib). I thought that it is worth copying the lib folder and to throw in the specified folder, but after I again decided to try to install rubygems, and here I began to write that I need to throw in a folder some exe file. But what I do not know, I do not even know what I did right that threw (lib) in this folder, because I'm new to this. Here is a picture showing the bug:https://imgur.com/a/9hDi3ef        
 P. S:I need rubygems for metasploit
